documentation/content/en/books/handbook/bsdinstall/_index.adoc

```diff
@@ -1068,23 +1068,23 @@ image::bsdinstall-mainexit.png[]
 If further configuration or special setup is needed, select btn:[Live CD] to boot the install media into Live CD mode.
 
 If the installation is complete, select btn:[Reboot] to reboot the computer and start the new FreeBSD system.
-Do not forget to remove the FreeBSD install media or the computer may boot from it again.
+Do not forget to remove the FreeBSD install media or the computer might boot from it again.
 
 As FreeBSD boots, informational messages are displayed.
 After the system finishes booting, a login prompt is displayed.
 At the `login:` prompt, enter the username added during the installation.
 Avoid logging in as `root`.
-Refer to crossref:basics[users-superuser,“The Superuser Account”] for instructions on how to become the superuser when administrative access is needed.
+Refer to crossref:basics[users-superuser,The Superuser Account] for instructions on how to become the superuser when administrative access is needed.
 
-The messages that appeared during boot can be reviewed by pressing kbd:[Scroll-Lock] to turn on the scroll-back buffer.
+The messages that appear during boot can be reviewed by pressing kbd:[Scroll-Lock] to turn on the scroll-back buffer.
 The kbd:[PgUp], kbd:[PgDn], and arrow keys can be used to scroll back through the messages.
 When finished, press kbd:[Scroll-Lock] again to unlock the display and return to the console.
 To review these messages once the system has been up for some time, type `less /var/run/dmesg.boot` from a command prompt.
 Press kbd:[q] to return to the command line after viewing.
 
-If sshd was enabled in <<bsdinstall-config-serv>>, the first boot may be a bit slower as the system will generate the RSA and DSA keys.
+If sshd was enabled in <<bsdinstall-config-serv>>, the first boot might be a bit slower as the system generates RSA and DSA keys.
```

Comment:
"RSA and DSA" is no longer true by default. From /etc/rd.d/sshd:
> : ${sshd_rsa_enable:="yes"}
> : ${sshd_dsa_enable:="no"}
> : ${sshd_ecdsa_enable:="yes"}
> : ${sshd_ed25519_enable:="yes"}

But that may be overriden, so maybe be vague and use "SSH host keys".